homeassistant/components/geniushub/binary_sensor.py

```diff
@@ -10,22 +10,36 @@
 from .entity import GeniusDevice
 
 GH_STATE_ATTR = "outputOnOff"
-GH_TYPE = "Receiver"
+GH_TYPE_RECEIVER = "Receiver"
+GH_TYPE_SWITCH = "Smart Plug"
+GH_TYPE_ELECTRIC_SWITCH = "Electric Switch"
 
 
 async def async_setup_entry(
     hass: HomeAssistant,
     entry: GeniusHubConfigEntry,
     async_add_entities: AddEntitiesCallback,
 ) -> None:
-    """Set up the Genius Hub binary sensor entities."""
-
+    """Set up the Genius Hub binary sensor entities. A binary sensor (Receiver, smart plug or electric switch) is only a binary sensor if it not alone in a zone."""
```

Comment:
Please follow PEP8. The first line in a docstring is a header and should be a single sentence. After that an optional body may follow separated from the header by a blank line. Docstring lines should not be longer than 72 characters.